code/game/objects/structures/candybucket.dm

```diff
@@ -0,0 +1,82 @@
+/obj/structure/candybucket
+	name = "candy bucket"
+	icon = 'icons/obj/candybucket.dmi'
+	icon_state = "jackbucket_empty"
+	desc = "A thematically appropriate bucket for filling with candy or other goodies."
+	density = 1
+	anchored = 1
```

Comment:
I'm confused, are these not meant to be carried around?

Reply:
It's meant to be a bucket the house leaves outside with a "Please take one" sign, not a bucket a trick or treater carries. The original concept had Jack get made if you stole candy from him twice but I scrapped that whole bit.